Kernel/Memory/MemoryManager.cpp

```diff
@@ -791,7 +791,7 @@ ErrorOr<NonnullOwnPtr<Memory::Region>> MemoryManager::allocate_dma_buffer_page(S
 ErrorOr<NonnullOwnPtr<Memory::Region>> MemoryManager::allocate_dma_buffer_pages(size_t size, StringView name, Memory::Region::Access access, NonnullRefPtrVector<Memory::PhysicalPage>& dma_buffer_pages)
 {
     VERIFY(!(size % PAGE_SIZE));
-    dma_buffer_pages = TRY(allocate_contiguous_supervisor_physical_pages(size));
+    dma_buffer_pages = TRY(allocate_contiguous_user_physical_pages(size));
```

Comment:
After this change, there are no more users of supervisor physical pages. Can we remove the concept and rename "user physical pages" to just "physical pages"?

Reply:
This is not entirely correct, as the AHCI code still uses supervisor pages:
https://github.com/SerenityOS/serenity/blob/d728017578c18ebcf6af9a6da8182ee15a597f75/Kernel/Storage/ATA/AHCIPort.cpp#L25-L28
https://github.com/SerenityOS/serenity/blob/d728017578c18ebcf6af9a6da8182ee15a597f75/Kernel/Storage/ATA/AHCIPort.cpp#L38-L49

However, I am willing to remove these use cases as well to make sure we don't have this separation needlessly :)

Reply:
Ah, I didn't notice the AHCI code using it. Still, it would be nice to get rid of the separation if possible yes :)